content/300-guides/050-database/900-advanced-database-tasks/08-sql-views.mdx

```diff
@@ -73,6 +83,8 @@ Be aware of the following limitations when using views with Prisma:
 - You must _manually_ add each view as a model to the Prisma schema right now. Introspection does not add views to the schema currently.
 - Views must include a unique column - such as an ID.
 - The generated Prisma Client will include queries such `create`, `delete`, and `update`, even though you cannot perform these queries on a view.
+- Models created for views will be **deleted** from your schema the next time you run an introspection. This is because they **do not** correspond to the tables in the database.
+- Prisma Migrate will treat these views as tables and try to create them, this can lead to crashes. If you have view models, you **cannot** use Prisma Migrate with them.
```

Comment:
Nitpick: it should not lead to crashes, I think we can omit this.

Reply:
Cool beans